### ISSUE TITLE
CurseForge Suggestion

### ISSUE THREAD

**Issue by Silviyius** [moraroy/NonSteamLaunchers-On-Steam-Deck#51]
Not sure if its possible but adding a script to get curseforge standalone working with this. I know minecraft is already added but this would allow modded minecraft.

**Comment by moraroy** [comment]
@Silviyius you've given me an idea thank you! Quick question is curseforge compatability with this legacy/msi Java version of mine raft and dungeons?

**Comment by Silviyius** [comment]
Its legacy Java.  It just the mod client for Forge or fabric.

**Comment by moraroy** [comment]
@Silviyius would this program mod other games as well or is this just strictly minecraft?

**Comment by Silviyius** [comment]
https://www.curseforge.com/games

They have a whole list of games.  The linux verison only supports WOW

**Comment by moraroy** [comment]
@Silviyius dang that's crazy.. ok.. you've given me a really good idea about mods. I'll look into it.

**Comment by moraroy** [comment]
@Silviyius apparently curseforge is malware?

**Comment by Silviyius** [comment]
@moraroy there was some modders who got infected with malware. Full story here

https://www.windowscentral.com/gaming/curseforge-pc-game-mod-users-may-have-received-malware-as-part-of-a-recent-hack

**Comment by moraroy** [comment]
@Silviyius dang :/ is it worth adding still in your opinion i dont mod too much but you given me a great idea its hard to pass up im going to implement it tonight

**Comment by Silviyius** [comment]
@moraroy Id say so. Everybody gets hacked eventually. I can live without it. It's just a pain to install otherwise. Thanks for supporting the Linux and steam deck community!

**Comment by color-space** [comment]
No this is not necessary, out of all launchers curseforges launcher is the most unnecessary, don't waste time with this :)

**Comment by color-space** [comment]
Why was minecraft even added? Thats not really a launcher, unless we call the bootstrap process of every game a launcher now..

**Comment by moraroy** [comment]
@color-space do you have any more launchers you can recommend?

**Comment by Silviyius** [comment]
Well, I disagree with color space that Minecraft isn't the launcher, which is. I'm glad you added that is an option that makes it easier for everybody. Not everyone plays the same games, so well I'll have different opinions.

**Comment by moraroy** [comment]
@Silviyius maybe I'm missing something...I want to add cursforge, but it won't install for me I even tried through overwolf and that won't install for me, have you successfully tried to install this in proton?

**Comment by buzzsurfr** [comment]
@moraroy Have you tried using the Linux version of CurseForge? I've not tried it (yet) but I'm watching [this video](https://www.youtube.com/watch?v=08PXxVNnXIg) and they use the Linux version, which is able to add the addons to WoW without any problems.

**Comment by moraroy** [comment]
@buzzsurfr yeah I could add it but I was doing my best to add the windows version because it supports a lot more games... I've been told the Linux version only supports wow? Is that true?

**Comment by buzzsurfr** [comment]
Ahh, you're correct--CurseForge for Linux supports WoW only. ([Reference](https://www.curseforge.com/download/app#download-options)).

**Comment by moraroy** [comment]
@if you can figure out a way to install cursforge windows version I'll add it!

**Comment by Silviyius** [comment]
Just a heads up. Looks like they came out with a linux verison of this working on the device now which is a app image.  Its pretty Slick.

https://www.curseforge.com/download/app

No proton needed.
